src/main/java/com/icthh/xm/tmf/ms/communication/rules/businesstime/BusinessDayConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.icthh.xm.tmf.ms.communication.rules.businesstime;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+public class BusinessDayConfig {
+
+    private BusinessTimeConfig businessTime;
+
+    @Data
```

Comment:
suggest to use `@Value` instead of `@Data`annotation if possible as it keeps object immutable